pagopa/io-app
# feat: [IOCOM-880] SEND cancelled message, new DS

⚠️ This PR depends on #5706  ⚠️

## Short description
This PR adapts the cancelled SEND message details, for the new DS.
|Cancelled<br/>No paid payments|Cancelled<br/>Paid payments|
|-|-|
|![Simulator Screenshot - iPhone 15 - 2024-04-19 at 16 48 22](https://github.com/pagopa/io-app/assets/5150343/b20c7c74-1698-46a3-a66d-62142a07de3a)|![Simulator Screenshot - iPhone 15 - 2024-04-19 at 16 47 57](https://github.com/pagopa/io-app/assets/5150343/2bd63a5e-eca7-466b-bae6-11731ebd41fe)|

## List of changes proposed in this pull request
- `MessageCancelledContent` component to display the warning alert
- `MessageBottomMenu` adapted to include paid notice codes in the "More Info" bottom sheet
- `MessagePayments` adapted not show the disclaimer anymore (which is shown in `MessageCancelledContent`)

## How to test
Using the io-dev-api-server:
- for a SEND message that is not cancelled, no disclaimer should be shown, payments, attachments and F24 should appear and be selectable and the "More Info" bottom sheet should include all payment's notice-codes and entity-codes;
- for a cancelled SEND message with no payments (no unpaid nor paid), a disclaimer should appear with no reference to the payments (in its text), attachments should be disabled, no payments should appear, no F24 should appear, no notice codes should appear in the "More Info" bottom sheet
- for a cancelled SEND message with unpaid payments but not paid ones, a disclaimer should appear with a reference to the payments (in its text), attachments should be disabled, no payments should appear, no F24 should appear, no notice codes should appear in the "More Info" bottom sheet
- for a cancelled SEND message with paid payments (regardless of unpaid ones), a disclaimer should appear with a reference to the payments (in its text), attachments should be disabled, paid payments should appear, no F24 should appear, paid notice codes should appear in the "More Info" bottom sheet
